pybliometrics/utils/create_config.py

```diff
@@ -33,23 +33,36 @@ def create_config(config_dir: Optional[Union[str, Path]] = None,
 
     # Set authentication
     config.add_section('Authentication')
-    if keys:
-        if not isinstance(keys, list):
+
+    # Get keys and tokens
+    new_keys, new_tokens = None, None
+    if custom_keys:
+        if not isinstance(custom_keys, list):
             raise ValueError("Parameter `keys` must be a list.")
-        key = ", ".join(keys)
-        token = insttoken
-    else:
+        new_keys = ", ".join(custom_keys)
+    if custom_tokens:
+        if not isinstance(custom_tokens, list):
+            raise ValueError("Parameter `inst_tokens` must be a list of tuples. "\
+                             "E.g. [('key_1', 'token_1'), ('key_2', 'token_2')]")
+        new_tokens = ", ".join([f"{key}:{value}" for key, value in custom_tokens])
```

Comment:
I think an easier setup is simply to zip the keys and the tokens. For robustness, use the first `n = len(tokens)` keys. That means, `insttoken` can remain to be a simple list.

Reply:
Good idea! Much simpler